apps/nextjs-website/src/components/templates/CaseHistoryTemplate/CaseHistoryPageTemplate.tsx

```diff
@@ -66,12 +66,7 @@ const CaseHistoryPageTemplate = ({
             <PartRenderer part={part} />
           </EContainer>
         ) : (
-          <Box
-            key={index}
-            sx={{
-              marginY: 3,
-            }}
-          >
+          <Box key={index}>
             <PartRenderer part={part} />
           </Box>
         )
```

Comment:
You should add a new prop `verticalPadding` to <ProductsShowcase component with 5 as default value